jsnip/src/main/java/dev/marcelo/java/Caller.java

```diff
@@ -19,15 +19,15 @@ public int getSomeInt() {
   }
 
   public void setSomeInt(int someInt) {
-    this.someInt = Callee.someInt + someInt;
+    this.someInt = Callee.someInt + someInt + 1;
   }
 
   public void testAPI() {
     int k = getSomeInt();
-    DummyAPI api = new DummyAPI(k);
+    DummyAPI api = new DummyAPI(k+1);
     RandomAPI rapi = new RandomAPI();
     String s = rapi.toString();
-    setSomeInt(api.getNum());
+    setSomeInt(api.getNum()+1);
```

Comment:
[reviewpad-sig]: <>
[Link to Comment](localhost:9000/review/github.com/marcelosousa/java-snippets/pull/3#comment-690218ba-7fd0-4473-becc-90a57516dc45)